sdk/communication/azure-communication-callautomation/azure/communication/callautomation/_utils.py

```diff
@@ -134,7 +135,7 @@ def serialize_identifier(identifier: CommunicationIdentifier) -> Dict[str, Any]:
     try:
         request_model = {"raw_id": identifier.raw_id}
```

Comment:
```suggestion
        request_model: dict[str, Any] = {"raw_id": identifier.raw_id}
```

this change will help you here `request_model[identifier.kind] = dict(identifier.properties)  # type: ignore[assignment]`

the initial assignment makes `request_model` be `dict[str, str]`. When its assigned properties which is dict, mypy complains.